### ISSUE TITLE
some covers don't appear

### ISSUE THREAD

**Issue by SkqLiao** [janeczku/calibre-web#1205]
I added nearly 200 books. But their're three books appeared quite weird.
The cover didn't appear and there was only the title of the book.
like this:
[1.jpg](https://i.loli.net/2020/02/19/4tAVTghDRpowOen.png)
I pressed CTRL+F5 but nothing changed.
Also I tried again to edit its cover but nothing changed.
like this:
[2.jpg](https://i.loli.net/2020/02/19/nupboPcCZTfrNRS.png)
Then I thought it might be the epub file's problem. But I can open it by both the E-book viewer in calibre and calibre-web in browser.
I checked the hard disk and cover.jpg is there. like this:
[3.jpg](https://i.loli.net/2020/02/19/ZcrQ8lD35szS4HF.png)

It looks like that these three books have nothing different from the others. But they just could't show their cover.

**Comment by OzzieIsaacs** [comment]
I don't get the problem with the second book. What's wrong?
For the firt two books, it looks indeed like the cover couldn't be loaded. 
Could you please check:
1.) Enter the address of the cover picture, should be http://[serveradress/cover/id in browser. Is the picture now showing up?
2:) Check the developer console on reloading the page. Is the request to the cover answered "ok" or do you see a 404 or similar error?

**Comment by OzzieIsaacs** [comment]
Is the cover showing up in calibre (the desktop program) itself? -> no, problem during upload
Is the cover showing up after another reload of the book?
What happens if you change the title or author of the book, is the cover now showing up?

**Comment by SkqLiao** [comment]
> Is the cover showing up in calibre (the desktop program) itself? -> no, problem during upload
> Is the cover showing up after another reload of the book?
> What happens if you change the title or author of the book, is the cover now showing up?

Thanks, your question inspired me.
I found that although cover.jpg existed, its size was 0kb.
Then I just deleted it and uploaded the cover again, its size turned normal and could appear normally.
It's weird because I just tried in another book. I deleted its cover and created an empty file "cover.jpg". Then I entered the edit page and uploaded its real cover. However, the file was replaced by the new one. But this didn't happen in the last three book occasions.
